src/utils/EntryModel.cxxtest

```diff
@@ -630,6 +630,64 @@ TEST(EntryModelBoundedTest, SetMinMax)
     EXPECT_FALSE(em.empty());
 }
 
+TEST(EntryModelBoundedTest, SetMinMaxUnclamped)
```

Comment:
should we have a test where min..max does not include 0? I think that was the specific bug we needed to address.

Reply:
Good idea, I will add this.

Reply:
Added.